setup.py

```diff
@@ -71,21 +73,19 @@ def maybe_rm(path):
     keywords='scheduler nextdoor cron python',
     packages=find_packages(),
     include_package_data=True,
-    extras_require={
-    'python_version<"3.3"': ['funcsigs']
-    },
+    extras_require={'python_version<"3.3"': ['funcsigs']},
     tests_require=[
         'funcsigs',
         'mock == 1.1.2',
         'nose',
     ],
     test_suite='nose.collector',
     install_requires=[
-        'APScheduler == 3.0.0',
-        'SQLAlchemy == 1.0.0',
-        'future == 0.15.2',
-        'tornado == 4.3.0',
-        'python-dateutil == 2.2',
+        'APScheduler == 3.0',
+        'SQLAlchemy',
+        'future',
+        'tornado',
+        'python-dateutil',
```

Comment:
ndscheduler is used as a complete application.  I'm going to re-pin these versions.

On that topic, ndscheduler *only* works with APScheduler == 3.0.x.